src/Components/FolderMain.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import searchIcon from "../Assets/image/Search.png";
+import { FolderCard } from "./FolderCard";
+import { FolderMenu } from "./FolderMenu";
+import { useFolderName } from "../Hooks/useFolderName";
+import { useFolder } from "../Hooks/useFolder";
+import FolderTools from "./FolderTools";
+import "../Styles/FolderMain.css";
+
+export function FolderMain() {
+  const [menu, setMenu] = useState("전체");
+
+  const handleMenuChange = (newMenu, id) => {
+    setMenu(newMenu);
+    fetchFolder(id);
+  };
+
+  const folderNames = useFolderName();
+  const { folder, fetchFolder } = useFolder();
+
+  return (
+    <>
+      <div className="searchContainer">
+        <div className="search">
+          <input
+            className="searchInput"
+            placeholder="링크를 검색해 보세요."
+          ></input>
+          <div className="searchIcon">
+            <img src={searchIcon} className="searchImg" alt="검색 아이콘"></img>
+          </div>
+        </div>
+      </div>
+
+      <FolderMenu folderNames={folderNames} onMenuChange={handleMenuChange} />
+
+      <div className="titleContainer">
+        <div className="title">{menu}</div>
+        {menu !== "전체" && <FolderTools />}
+      </div>
+
+      {folder && folder.length ? (
```

Comment:
```suggestion
      {folder && folder.length >=1 ? (
```
그냥 length 가 있는지 여부보다 조금더 명시적으로 1보다 큰지를 적어주는게 좋을것 같아요!